### ISSUE TITLE
Load a checkpoint after warmup

### ISSUE THREAD

**Issue by Felonious-Spellfire** [Lightning-AI/lightning#14531]
## 📚 Documentation

https://pytorch-lightning.slack.com/team/U01H0HH3USF

Set's say I load a checkpoint, but then want to overwrite the learning rate schedulers. I'm using [pl_bolts.optimizers.lr](http://pl_bolts.optimizers.lr/)_scheduler.LinearWarmupCosineAnnealingLR which is deprecated :disappointed:

But basically it take 10 hours to do warmup. What I want to do is load the checkpoint from after warmup, and then just play with my cosine annealing scheduler. How can I do this? (I am worried there might be secret state in Lightning that I am not aware of.)

**Comment by rohitgr7** [comment]
I think @otaj is working on checkpoint splitting. probably related to this?

**Comment by otaj** [comment]
Yes, I am working on checkpoint splitting, however, other things had a slightly higher priority so I don't really have much to show for at the moment. However, the idea is simple: if we allow to split the checkpoint and user then *explicitly removes* some parts of that checkpoint (such as LR state), it will be missing from reconstructed checkpoint and therefore not loaded.
